src/garage/torch/policies/base.py

```diff
@@ -0,0 +1,53 @@
+
+
+class Policy:
+    """
+    Policy base class without Parameterzied.
+
+    Args:
+        env_spec (garage.envs.env_spec.EnvSpec): Environment specification.
+
+    """
+    def __init__(self, env_spec):
+        self._env_spec = env_spec
+        self._variable_scope = None
```

Comment:
Hmm there is no variable scope in pytorch..

Reply:
I will remove the _variable_scope